tests/integration/pdo/mysql/base-class/test_prepared_stmt_1.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*DESCRIPTION
+When PDO base class constructor is used to create connection object
+and a query is executed via PDOStatement::execute(), the agent should
+ - not generate errors
+ - record a datastore metrics
+ - record a datastore span event
+Moreover, when the query execution time exceeds the explain threshold,
+the agent should record a slow sql trace with explain plan.
+*/
+
+/*SKIPIF
+<?php require(realpath (dirname ( __FILE__ )) . '/../../skipif_mysql.inc');
+*/
+
+/*ENVIRONMENT
+DATASTORE_PRODUCT=MySQL
+DATASTORE_COLLECTION=tables
+*/
+
+/*INI
+;comment=Set explain_threshold to 0 to ensure that the slow query is recorded.
```

Comment:
I wanted the comments explaining these settings to be as close to the settings as possible. The reason for `;comment=...` construct is because `integration_runner` expects `key=value` format for anything (including comments) in `INI` directive.